Tests/OpenCombineTests/FoundationTests/OperationQueueSchedulerTests.swift

```diff
@@ -5,6 +5,10 @@
 //  Created by Sergej Jaskiewicz on 14.06.2020.
 //
 
+// OperationQueue has serious bugs in swift-corelibs-foundation prior to Swift 5.3.
+// (see https://github.com/apple/swift-corelibs-foundation/pull/2779)
+#if canImport(Darwin) || (swift(>=5.3) && os(Linux)) // TEST_DISCOVERY_CONDITION
+
```

Comment:
```suggestion
// OperationQueue has serious bugs in swift-corelibs-foundation prior to Swift 5.3.
// (see https://github.com/apple/swift-corelibs-foundation/pull/2779)
#if canImport(Darwin) || swift(>=5.3) && !WASI // TEST_DISCOVERY_CONDITION

```

Please also update the comment after `#endif`.